src/main/java/RedisSessionManagerT8.java

```diff
@@ -0,0 +1,298 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+import org.apache.juli.logging.Log;
+import org.apache.juli.logging.LogFactory;
+
+/**
+ * Created by Cesar Valverde on 6/10/2015.
+ */
+public class RedisSessionManagerT8 extends ManagerBase implements Lifecycle {
+
+    private final Log log = LogFactory.getLog(RedisSessionManagerT8.class);
+
+    private static final String REDIS_METADATA_KEY = ":metadata";
+    private static final String REDIS_ATTRIBUTES_KEY = ":attributes";
+
+    //Metadata keys
+    public static final String METADATA_VALID = "valid";
+    public static final String METADATA_CREATION_TIME = "creation_time";
+    public static final String METADATA_LAST_ACCESS_TIME = "last_access_time";
+    public static final String METADATA_MAX_INACTIVE_INTERVAL = "max_inactive_interval";
+
+    protected JedisPool redisConnectionPool = null;
+    protected JedisPoolConfig redisConnectionPoolConfig = null;
+
+    //Database info
+    private String dbHost = "localhost";
+    private int dbPort = 2222;
+    private int dbTimeout = 0;
+    private String dbPassword = "";
+
+    private static final String ISO_DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ssZ";
+
+
+    /**
+     * The lifecycle event support for this component.
+     */
+    protected LifecycleSupport lifecycle = new LifecycleSupport(this);
+
+    @Override
+    public void load() throws ClassNotFoundException, IOException {
+    }
+
+    @Override
+    public void unload() throws IOException {
+    }
+
+    //------------------------------------------------------------------------------------------------------------------
+    // START Sessions
+    //------------------------------------------------------------------------------------------------------------------
+
+    @Override
+    public int getRejectedSessions() {
+        // Essentially do nothing.
+        return 0;
+    }
+
+    @Override
+    public Session createSession(String requestedSessionId) {
+
+        Session session = createEmptySession();
+
+        // Initialize the properties of the new session and return it
+        session.setId(UUID.randomUUID().toString());
+        session.setNew(true);
+        session.setValid(true);
+        session.setCreationTime(System.currentTimeMillis());
+        session.setMaxInactiveInterval(this.maxInactiveInterval);
+
+        try {
+            Jedis jedis = redisConnectionPool.getResource();
+            saveToRedis(jedis, session);
+            redisConnectionPool.returnResourceObject(jedis);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        return session;
+    }
+
+    /**
+     * Add session to this manager
+     * @param session
+     */
+    @Override
+    public void add(Session session) {
+        try {
+            Jedis jedis = redisConnectionPool.getResource();
+            saveToRedis(jedis, session);
+            redisConnectionPool.returnResourceObject(jedis);
```

Comment:
You have repeated this exact code twice (82-84 and here). Refactor this so you aren't repeating yourself.